app/src/main/res/layout/component_history.xml

```diff
@@ -25,8 +26,17 @@
         layout="@layout/recently_closed_nav_item"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
+        android:layout_marginTop="8dp"
         app:layout_constraintTop_toTopOf="parent" />
 
+    <include
+        android:id="@+id/synced_history_nav_empty"
+        layout="@layout/synced_history_nav_item"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="8dp"
```

Comment:
There doesn't seem to be any additional spacing between list items. Can you send a screenshot of the figma inspection where you are seeing this 8dp in between the items?
<img width="418" alt="Screen Shot 2022-04-06 at 6 44 02 PM" src="https://user-images.githubusercontent.com/1190888/162084668-3b1f560e-4334-4a77-9f82-76fcd0037c91.png">

Reply:
If i am understanding this correctly, I think you just want a Spacer view in after the ProgressBar, but please double check with UX if this 8dp spacing from the top app bar is intended.

Reply:
If this proves too controversial, I would move this 8dp spacer into a followup because it's affecting a lot of your XML and that is gonna also affect the tab history view, which is when you hold on to the back arrow.

Reply:
Good catch! this depends a little on what we do on bookmarks and downloads; it would be great to add 8dp spacing from top bar to first item but then also have it consistent with the other "Library" screens; if we don't have the spacing there we can ignore it for now and I file a follow up, thanks!

Reply:
It's not part of the new design, it's to match the existing one: the populated list has extra spacing, but the empty state hasn't. it results into `recentlyClosedTabs` and  `synced_history_nav_item` items jumping when you change between empty and not empty state

Reply:
> Good catch! this depends a little on what we do on bookmarks and downloads; it would be great to add 8dp spacing from top bar to first item but then also have it consistent with the other "Library" screens; if we don't have the spacing there we can ignore it for now and I file a follow up, thanks!

We actually have this extra spacing in History screen when there are elements inside the list, but don't have it when the list is empty. I added the spacing to match the populated state of the screen.